app/enterprise/0.34-x/installation/docker.md

```diff
@@ -59,17 +54,17 @@ is required**:
       postgres:9.5
     ```
 
-8. To make the license data easier to handle, export it as a shell variable.
+7. To make the license data easier to handle, export it as a shell variable.
 Please note that **your license contents will differ**! Users with Bintray
-accounts should visit [https://bintray.com/kong/&lt;YOUR_REPO_NAME&gt;/license#files](https://bintray.com/kong/<YOUR_REPO_NAME>/license#files)
+accounts should visit `https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`
 to retrieve their license. Trial users should download their license from their
 welcome email. Once you have your license, you can set it in an environment variable:
 
     ```sh
     export KONG_LICENSE_DATA='{"license":{"signature":"LS0tLS1CRUdJTiBQR1AgTUVTU0FHRS0tLS0tClZlcnNpb246IEdudVBHIHYyCgpvd0did012TXdDSFdzMTVuUWw3dHhLK01wOTJTR0tLWVc3UU16WTBTVTVNc2toSVREWk1OTFEzVExJek1MY3dTCjA0ek1UVk1OREEwc2pRM04wOHpNalZKVHpOTE1EWk9TVTFLTXpRMVRVNHpTRXMzTjA0d056VXdUTytKWUdNUTQKR05oWW1VQ21NWEJ4Q3NDc3lMQmorTVBmOFhyWmZkNkNqVnJidmkyLzZ6THhzcitBclZtcFZWdnN1K1NiKzFhbgozcjNCeUxCZzdZOVdFL2FYQXJ0NG5lcmVpa2tZS1ozMlNlbGQvMm5iYkRzcmdlWFQzek1BQUE9PQo9b1VnSgotLS0tLUVORCBQR1AgTUVTU0FHRS0tLS0tCg=","payload":{"customer":"Test Company Inc","license_creation_date":"2017-11-08","product_subscription":"Kong Enterprise","admin_seats":"5","support_plan":"None","license_expiration_date":"2017-11-10","license_key":"00141000017ODj3AAG_a1V41000004wT0OEAU"},"version":1}}'
     ```
 
-9. Run Kong migrations:
+8. Run Kong migrations:
```

Comment:
@levfishbluefish as discussed, this might clash with passing KONG_PASSWORD, which is in the Getting Started guide:

https://docs.konghq.com/enterprise/0.34-x/getting-started/quickstart/#1-start-kong-enterprise

We don't want them to run migrations here first, only to find the same steps later.

Reply:
If we remove this then we will have to note that Kong will be empty until they follow the getting started guide, which may be confusing